priv/templates/boot_win.eex

```diff
@@ -2,6 +2,8 @@
 ::
 :: Check the :usage section for information about what this script provides
 
+@echo off
```

Comment:
Could you explain why you added this? I'm only asking because it's been awhile since I wrote the batch script, and I want to make sure I understand the changes being made.